bodhi/tests/server/functional/test_generic.py

```diff
@@ -101,6 +103,81 @@ def test_home(self):
         self.assertIn('Log out', res)
         self.assertIn('Fedora Update System', res)
 
+    def test_home_counts(self):
```

Comment:
It would be good to add a docblock saying that this test is making sure that the home page update count report works.